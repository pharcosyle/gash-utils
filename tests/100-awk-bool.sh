echo -e "one\ntwo" | awk -f tests/data/awk/bool.awk
