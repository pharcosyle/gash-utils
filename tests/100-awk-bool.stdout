unset:
1:isFALSE
0;TRUE
one
unset:0
1:isFALSE
0;TRUE
two
